integration/cli-tests/src/test/java/org/wso2/micro/integrator/cli/Constants.java

```diff
@@ -20,13 +20,16 @@
 
 public class Constants {
 
+    public static final String LOGIN = "login";
+    public static final String DEFAULT_USERNAME = "admin";
+    public static final String DEFAULT_PASSWORD = "admin";
     static final String API = "api";
     static final String SHOW = "show";
     static final String COMPOSITAPP = "compositeapp";
     static final String ENDPOINT = "endpoint";
     static final String PROXYSERVICE = "proxyservice";
     static final String SEQUENCE = "sequence";
-    static final String REMOTE = "remote";
+    public static final String REMOTE = "remote";
```

Comment:
Any reason to have only some constants as public? Or is being package private enough?